tests/unit_tests/test_region.py

```diff
@@ -34,6 +34,12 @@ def test_union(reset):
     assert (6, -1, 0) not in reg2
     assert str(reg2) == '((1 | -2) 3)'
 
+    # translate method
+    regt = region.translate((2.0, 0.0, 0.0))
+    assert (-4, 0, 0) in regt
```

Comment:
The `translate` methods have if/else blocks where if the translation functionally does nothing, return self. Should we check for those in the tests too?

Reply:
I didn't see it as absolutely necessary. It's really more an of optimization rather than expected behavior that must be relied on (i.e., tested). However, if you feel differently, I can add such checks.

Reply:
Nah, thats a good point of view.  Not necessary until we try and score 100% on test coverage.